idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
         \type object-list
         \object-list DesignSpecificationZoneHVACSizingName
 
+ZoneHVAC:HybridUnitaryHVAC,		
+		\memo Hybrid Unitary HVAC. A black box model for multi-mode packaged forced air equipment. Independent variables include outside air conditions and indoor air conditions. Controlled inputs include operating mode, supply air flow rate, and outside air faction. Emperical lookup tables are required to map supply air temperature supply air humidity, electricity use, fuel uses, water use, fan electricity use, and external static pressure as a function of each indpednent varaible and each controlled input. In each timestep the model will choose one or more combinations of settings for mode, supply airflow rate, outside air faction, and part runtime fraction so as to satisfy zone requests for sensible cooling, heating, ventilation, and/or dehumidification with the least resource consumption. Equipment in this class may consume electricity, water, and up to two additional fuel types.
+	A1,	\field Name
+		\required-field
+		\type alpha
+		\retaincase
+        \reference ZoneEquipmentNames
+	A2,	\field Availability Schedule Name
+		\note Enter the availability schedule name for this system. Schedule value > 0 means the system is available. If this field is blank, the system is always available.
```

Comment:
Line length comment goes for \note also.  I believe the soft max we use is 100 characters.